etc/configure.py

```diff
@@ -261,6 +272,9 @@ def create_virtualenv(std_python, root_dir, tpp_dirs=(), quiet=False):
         vcmd += ['-qq']
     # third parties may be in more than one directory
     vcmd.extend(build_pip_dirs_args(tpp_dirs, root_dir))
+    # Window doesn't support link as extra-search-dir
```

Comment:
Is there a link  in the doc or a bug about this? Please add it there as a comment

Reply:
i didn't get docs about this specifying same issue.